scripts/ci_build_and_test_in_container.sh

```diff
@@ -256,8 +280,14 @@ if [[ "${RUN_INTEGRATED_TESTS}" = true ]]; then
   or_die tar cfM ${DATA_EXCHANGE_DIR}/${DATA_BASENAME_WE}.tar --directory ${GEOS_SRC_DIR}    --transform "s/^integratedTests/${DATA_BASENAME_WE}\/repo/" integratedTests
   or_die tar rfM ${DATA_EXCHANGE_DIR}/${DATA_BASENAME_WE}.tar --directory ${GEOSX_BUILD_DIR} --transform "s/^integratedTests/${DATA_BASENAME_WE}\/logs/" integratedTests
   or_die gzip ${DATA_EXCHANGE_DIR}/${DATA_BASENAME_WE}.tar
+
+  # want to clean the integrated tests folder to avoid polluting the next build.
+  or_die integratedTests/geos_ats.sh -a clean
```

Comment:
With @cssherman 's patch to build the baselines out of source, cleaning the integrated tests will surely not be mandatory anymore, since they disappear with the `docker run --rm`.
Also, we will be able to mount the source `read-only`!

Reply:
I think this was before the `--rm` command. I can try without.

Reply:
Oh, I think @cssherman has not been merged already. Am I wrong?

Reply:
`clean` not `veryclean` ?  I think it will still be better to clean up the integratedTests directory. One may update the version of the baselines... and the clean also removes the tests results which I think we do want to remove IMO.

Reply:
An option can be to mount the workspace read-only in the container, then copy it to a temporary and work in it (waiting for an out-of-source integrated tests which should solve the last item which touches the src dir).
Then everything would be cleaned by the `--rm`.

Eventually, we'll be able to skip the workspace copy, but not right now.